### PR TITLE
Description field added to policy groups

### PR DESCRIPTION
**Pull request by lahiru823**

### CODE REVIEW

File: components/appmgt/org.wso2.carbon.appmgt.api/src/main/java/org/wso2/carbon/appmgt/api/model/WebApp.java [hunk 1]
```diff
@@ -100,7 +102,7 @@
     private String uuid;
     private String policyPartials;
     private String policyGroups; //Policy Groups Id's list
-    private String javaPolicies; //Java policies(handlers) List
+    private JSONArray javaPolicies; //Java policies(handlers) List
```
Comment (rushmin) [inline components/appmgt/org.wso2.carbon.appmgt.api/src/main/java/org/wso2/carbon/appmgt/api/model/WebApp.java:105]: You can simply replace the JSONArray with a Java List. It's a bad idea to depend on third party data structures when there are inbuilt data structures.

File: components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java [hunk 2]
```diff
@@ -6554,12 +6554,13 @@ public static JSONArray getAppsByHitCount(String userId,
 	 * @param userRoles          : user roles
 	 * @param isAnonymousAllowed : is anonymous access allowed to URL pattern
 	 * @param objPartialMappings : Object which contains XACML policy partial details arrays
+	 * @param policyGroupDesc    :Policy group Desciption
 	 * @return : last saved policy group id
 	 * @throws AppManagementException if any an error found while saving data to DB
 	 */
 	public static Integer savePolicyGroup(String policyGroupName, String throttlingTier,
```
Comment (rushmin) [inline components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java:6561]: Too many parameters. A 'PolicyGroup' type can be introduced.

File: components/appmgt/org.wso2.carbon.appmgt.api/src/main/java/org/wso2/carbon/appmgt/api/model/WebApp.java (hunk 1 above)
Comment (lahiru823) [inline components/appmgt/org.wso2.carbon.appmgt.api/src/main/java/org/wso2/carbon/appmgt/api/model/WebApp.java:105]: In the latest code type has been changed to "String" to except comma separated ID list

File: components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java (hunk 2 above)
Comment (lahiru823) [inline components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java:6561]: Added a jira https://wso2.org/jira/browse/APPM-1072
Similar issue will be fixed through out the code in future releases.

**Pull request merged**